Silicon/NXP/iMX6Pkg/Include/common_epit.h

```diff
@@ -0,0 +1,168 @@
+/** @file
+*
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*
+*  This program and the accompanying materials
+*  are licensed and made available under the terms and conditions of the BSD License
+*  which accompanies this distribution.  The full text of the license may be found at
+*  http://opensource.org/licenses/bsd-license.php
+*
+*  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+*
+**/
+//  Header:  common_epit.h
+//
+//  Provides definitions for the EPIT (Enhanced Periodic Interrupt Timer)

```

Comment:
2 license headers with different styles looks ugly. You just add a line for Freescale below Copyright (c) Microsoft Corporation. All rights reserved.

Reply:
The same applies to the other header as well.